components/Card/index.js

```diff
@@ -80,6 +81,18 @@ export default function Card({
         </CardBody>
       </Section>
       <CardFooter>
+        {" "}
```

Comment:
You can remove empty lines like this one (if you don't need them of course). They've likely been put here by formatting.